exercises/grains/README.md

```diff
@@ -33,7 +33,7 @@ For installation and learning resources, refer to the
 Run the tests by executing the test script.
 
 ```
-$ groovy ./HelloWorldSpec.groovy
+$ groovy ./GrainsSpec.groovy
```

Comment:
Did this get generated via the `configlet generate . --only grains` command?

If not it looks like we need to open an issue to fix the template so that it uses the name of the exercise in the code example.